packages/manager/modules/anthos/src/dashboard/general-information/general-information.html

```diff
@@ -0,0 +1,235 @@
+                data-with-engagement="$ctrl.isCommitmentAvailable"
+                data-highlight-engagement="$ctrl.shouldReengage"
+                data-go-to-commit="$ctrl.goToCommit()"
+                data-go-to-cancel-commit="$ctrl.goToCancelCommit()"
+                data-go-to-cancel-resiliation="$ctrl.goToCancelResiliation()"
+                data-go-to-resiliation="$ctrl.goToResiliation()"
+                data-on-error="$ctrl.onBillingInformationError(error)"
+                data-tracking-prefix="{{ ::$ctrl.trackingPrefix }}"
+                data-commit-impression-data="$ctrl.shouldReengage ?
+                    $ctrl.RECOMMIT_IMPRESSION_TRACKING_DATA :
+                    $ctrl.COMMIT_IMPRESSION_TRACKING_DATA"
+            >
+            </billing-subscription-tile>
+        </div>
+
+        <!--Resources tile-->
+        <div class="col-md-4">
+            <oui-tile
+                data-heading="{{:: 'anthos_tenant_dashboard_general_information_tile_resources_title' | translate }}"
+            >
+                <!--Hosts-->
+                <oui-tile-definition
+                    data-term="{{:: 'anthos_tenant_dashboard_general_information_tile_resources_hosts' | translate }}">
+                    <oui-action-menu aria-label="Actions" data-compact data-placement="end">
+                        <oui-action-menu-item data-on-click="$ctrl.goToOrderHost()">
+                            <span
+                                data-translate="anthos_tenant_dashboard_general_information_tile_resources_hosts_menu_order_host"
+                            ></span>
+                        </oui-action-menu-item>
+                    </oui-action-menu>
+
+                    <oui-tile-description>
+                        <span data-ng-bind="$ctrl.hosts.length"></span>
+                        <span> </span>
+                        <span data-translate="anthos_tenant_dashboard_general_information_tile_resources_hosts_quantity"></span>
+                    </oui-tile-description>
+                </oui-tile-definition>
+
+                <!--Persistent storage-->
+                <oui-tile-definition
+                    data-term="{{:: 'anthos_tenant_dashboard_general_information_tile_resources_persistent_storage' | translate }}"
+                    data-description="{{:: $ctrl.tenant.storage }}"
+                >
+                    <oui-tile-description>
+                        <p class="mb-1">
+                            <span data-ng-bind=":: $ctrl.netappStorage.totalSize | cucBytes:0:false:'MB' "></span>
+                        </p>
+                        <p class="mb-1">
+                            <span data-ng-bind=":: $ctrl.getStorageInfo()"></span>
+                        </p>
+                        <p class="mb-1">
+                            <oui-progress>
+                                <oui-progress-bar data-type="info"
```

Comment:
Please try to re-use the component from https://github.com/ovh/manager/pull/5440/files#diff-ac476601f5490298d5b44f655f2e645d3295111c09faac8aafc0396f5bd08399

Reply:
@ganeshkumar1989 , to avoid blocking review, we can plan to reuse the component an another task.

Reply:
Please create a task and reference the task id here, in order to track it please